samples/matter/window_covering/README.rst

```diff
@@ -113,60 +113,86 @@ Factory data support
 User interface
 **************
 
-.. include:: ../template/README.rst
-   :start-after: matter_template_nrf54l15_0_3_0_interface_start
-   :end-before: matter_template_nrf54l15_0_3_0_interface_end
+.. tabs::
 
-.. include:: ../lock/README.rst
-    :start-after: matter_door_lock_sample_led1_start
-    :end-before: matter_door_lock_sample_led1_end
+   .. group-tab:: nRF52 and nRF53 DKs
 
-LED 2
-    Indicates the lift position of the window cover, which is represented by the brightness of the LED.
-    The brightness level ranges from ``0`` to ``255``, where the brightness level set to ``0`` (switched off LED) indicates a fully opened window cover (lifted) and the brightness level set to ``255`` indicates a fully closed window cover (lowered).
+      LED 1:
+         .. include:: /includes/matter_sample_state_led.txt
 
-    Additionally, the LED starts blinking evenly (500 ms on/500 ms off) when the Identify command of the Identify cluster is received on the endpoint ``1``.
-    The command's argument can be used to specify the duration of the effect.
+      LED 2:
+         Indicates the lift position of the window cover, which is represented by the brightness of the LED.
+         The brightness level ranges from ``0`` to ``255``, where the brightness level set to ``0`` (switched off LED) indicates a fully opened window cover (lifted) and the brightness level set to ``255`` indicates a fully closed window cover (lowered).
 
-LED 3 (nRF52840 DK and nRF5340 DK)
-    Indicates the tilt position of the window cover, which is represented by the brightness of the LED.
-    The brightness level ranges from ``0`` to ``255``, where the brightness level set to ``0`` (switched off LED) indicates a fully opened window cover (tilted to a horizontal position) and the brightness level set to ``255`` indicates a fully closed window cover (tilted to a vertical position).
+         Additionally, the LED starts blinking evenly (500 ms on/500 ms off) when the Identify command of the Identify cluster is received on the endpoint ``1``.
+         The command's argument can be used to specify the duration of the effect.
 
-LED 4 (nRF54L15 PDK)
-    .. note::
-       This is **LED 3** on the board revision v0.3.0.
+      LED 3:
```

Comment:
All throughout this sample LED 3 is referred in the same places for both nRF54DKs and other DKs, is this intended?

Reply:
This one is intended. Functionality required by the sample is not available to all LEDs in the current nRF54L15 PDKs.

Reply:
got it, thanks!